bin/fuel-core/src/cli/run/relayer.rs

```diff
@@ -15,32 +15,32 @@ use std::str::FromStr;
 #[derive(Debug, Clone, Args)]
 pub struct RelayerArgs {
     /// Enable the Relayer. By default, the Relayer is disabled, even when the binary is compiled
-    /// with the "relayer" feature flag. Providing `--enable-relayer` will enable the relayer
+    /// with the "Relayer" feature flag. Providing `--enable-relayer` will enable the Relayer
```

Comment:
I'm not sure that "relayer" should be capitalized. Maybe add it to the list in lowercase?